IndomitableIbises/Bangazon-Sprint-2
# Si training testing

## Link to Ticket
Closes # [Link]()

## Description of Proposed Changes

- added paths for training
-added basic tests for training

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout SI_training_testing
```
kill the DB and migrations
python manage.py makemigrations bang
python manage.py migrate
add a training to the db
python manage.py runserver
open the training page
should show it on a list element
python manage.py test should run a test and give you an ok

## Impacted Areas in Application

List general components of the application that this PR will affect:
url.py
training.html
nav_training.html
test.py
I think that's all?
*

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
